order-book/src/main/java/com/market/banica/order/book/service/grpc/OrderBookService.java

```diff
@@ -64,35 +47,39 @@ public void getOrderBookItemLayers(ItemOrderBookRequest request, StreamObserver<
 
     @Override
     public void announceItemInterest(InterestsRequest request, StreamObserver<InterestsResponse> responseObserver) {
+        final String itemName = request.getItemName();
 
         try {
 
-            auroraClient.startSubscription(request.getItemName(), request.getClientId());
+            auroraClient.startSubscription(itemName, request.getClientId());
```

Comment:
probably the request.clientId should be also extracted as the itemName

Reply:
Fixed